independent-publisher-connectors/Mintlify/apiDefinition.swagger.json

```diff
@@ -0,0 +1,198 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Mintlify",
+    "description": "Hate writing code documentation? Save time and improve your codebase by letting Mintlify generate documentation for you.",
+    "version": "1.0",
+    "contact": {
+      "name": "Troy Taylor",
+      "url": "https://www.hitachisolutions.com",
+      "email": "ttaylor@hitachisolutions.com"
+    }
+  },
+  "host": "api.mintlify.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": ["application/json"],
+  "produces": ["application/json"],
+  "paths": {
+    "/v1/document": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "documentation": {
+                  "type": "string",
+                  "description": "The documentation.",
+                  "title": "Documentation"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Generate documentation",
+        "description": "Returns a documentation string based on the code submitted.",
+        "operationId": "DocGen",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "commented": {
+                  "type": "boolean",
+                  "description": "Whether or not to comment the documentation output.",
+                  "title": "Commented",
+                  "default": true,
+                  "enum": [
+                    true,
+                    false
+                  ]
+                },
+                "language": {
+                  "type": "string",
+                  "description": "The programming language of the code.",
+                  "title": "Language",
+                  "default": "javascript",
+                  "enum": [
+                    "javascript",
+                    "typescript",
+                    "javascriptreact",
+                    "typescriptreact",
+                    "python",
+                    "php"
+                  ]
+                },
+                "code": {
+                  "type": "string",
+                  "description": "The code to generate documentation for.",
+                  "title": "Code"
+                },
+                "format": {
+                  "type": "string",
+                  "description": "The format of the documentation. Only works if commented is true.",
+                  "title": "Format",
+                  "enum": [

```

Comment:
I could be wrong, but I thought default values were only needed for required parameters. In this case, format is optional because there is another parameter that must be true for this to not error.